File: prob36/prob36.js
function countZeros(array) {
	var count = 0;
	for (var i = 0; i < array.length; i++) {
		array[i] === 0 ? count++ : null;
	}
	return count;
}

console.log(countZeros([0,1,0,2,3]))
console.log(countZeros([0,1,0,0,0]))
console.log(countZeros([0]))

File: prob36/prob36.py
def countZeros(stri):
    count=0

    for i in stri:
        if stri[i] == 0:
            count=count + 1
        else:
            count=count + 0
    return count

print "{0}".format(countZeros([0,0,0,1,1,2,4,0]))
